config/config.go

```diff
@@ -103,6 +105,7 @@ func init() {
 		Cfg.Rewards = core_config.MakeRewards(core_config.StageClaimAuthorities, core_config.StageRewardExtensions)
 		Cfg.AudiusdURL = "creatornode11.staging.audius.co"
 		Cfg.ChainId = "audius-testnet-alpha"
+		Cfg.AAOServer = "https://discoveryprovider.staging.audius.co"
```

Comment:
we already has
https://github.com/AudiusProject/api/blob/6a3f0b669ab3524e6f54a37967a0aed2cecc425d/config/config.go#L97